spec/views/content_items/show.html.erb_spec.rb

```diff
@@ -60,6 +60,14 @@
     expect(rendered).to have_selector('td + td', text: '2 months ago')
   end
 
+  it 'renders a link to FeedEx' do
+    content_item.base_path = '/the-base-path'
+    feedex_link = "https://support.publishing.service.gov.uk/anonymous_feedback?path=/the-base-path"
+    render
+
+    expect(rendered).to have_link('View anonymous feedback on FeedEx', href: feedex_link)
```

Comment:
Sorry @pmanrubia I was thrown off a bit as the current [Trello card](https://trello.com/c/kUebR1t4/184-2-show-content-feedback-link-to-feedex) said to use the URL to anonymous contacts in FeedEx, so I'll amend this.